app/views/spree/users/_api_keys.html.haml

```diff
@@ -0,0 +1,7 @@
+%hr
+%h3= t('spree.users.developer_settings.api_keys.title')
+%br
+%p
+  = text_field_tag :api_key, @user.spree_api_key, disabled: true, class: 'title'
+= form_tag spree.api_keys_path, method: :post, class: 'inline' do
+  = button_tag(t('spree.api.regenerate_key'), type: 'submit', class: "button primary")
```

Comment:
Out convention is to use lazy lookup:
```suggestion
  = button_tag(t('.regenerate_key'), type: 'submit', class: "button primary")
```

You will need to move the translation to `en.spree.users.api_keys.regenerate_key` then.

Reply:
Good to know! :thumbs_up: